exotations/collision_scene_fcl_latest/CMakeLists.txt

```diff
@@ -3,21 +3,28 @@ project(collision_scene_fcl_latest)
 
 add_compile_options(-std=c++11)
 
+
 find_package(catkin REQUIRED COMPONENTS
```

Comment:
Why does this have to be a two-step find_package catkin?

Reply:
Actually, I suspect this is to get a different order of multiple FCL include dirs in the array of include dirs, because the catkin-found one is different from the one that comes through exotica, correct? Perhaps add an explanatory comment

Reply:
I needed to compile against the exotica headers but not link against its libraries. Only the last `find_package` is used for linking.
The first one extracts the exotica header paths and the second one sets the linked libraries. I didn't want to link against exotica as that would link the the wrong version of FCL. I'm not entirely sure if this solves the problem once the plugin is loaded, but the SO file has a lot fewer dependencies (checked through `ldd`).